### ISSUE TITLE
Sample output of imagenet (CaffeNet) training?

### ISSUE THREAD

**Issue by cuihenggang** [BVLC/caffe#2707]
Hi,

I'm trying to train the CaffeNet model on imagenet dataset, using the default solver.prototxt and train_val.prototxt file.

However, I find the loss output during the training never goes below 6.9. Maybe I just haven't run long enought yet, but I'm wondering how many hours does it take to converge? Can someone share with me their LOG so that I will know whether my training is running correctly.

Thanks,
Cui

**Comment by seanbell** [comment]
CaffeNet takes about a week to train on a modern GPU from scratch, depending on your initialization and dataset.  If you're fine-tuning from a net trained on ILSVRC, you can train it in less than a day.

If you're using CPUs only, you want to use OpenBlas to enable multi-threaded matrix computations, or else it'll take forever (I've never tried it but I'm guessing several weeks at least).

Unless you're changing the architecture, you almost always want to fine-tune from the ImageNet weights.  In nearly all cases it both trains faster and reaches a better final performance.  The loss should also start decreasing right away.

Please ask these kinds of questions on the [caffe users group](https://groups.google.com/forum/#!forum/caffe-users).

**Comment by shelhamer** [comment]
The caffe-users group is the right place for this discussion, thanks.
